IR-docs-v4 (#341)
mattermost/mattermost-plugin-incident-response: docs/playbooks/creating-and-managing-playbooks.md

Within each playbook, you can create stages and tasks to ensure that items are addressed and completed in sequential order. The tasks can optionally be associated with slash commands and assigned to individual team members.

## Creating a Playbook

A playbook must be defined before starting an incident.

1. Navigate to **Main Menu > Playbooks & Incidents**.
2. Select a template, or **+ Create a Playbook** to start a new playbook from scratch.
4. Name your playbook.
5. Edit the **Default Stage**, defining one or more steps to be taken by members of the incident.
   * Optionally use descriptions on steps to add additional context for members of the incident. Descriptions support a limited form of markdown, including text styling and hyperlinks.
   * Optionally define a slash command with the step, simplifying the completion of steps in the incident.
6. Configure whether the incident channel should be public or private within the team.
7. Share this playbook with other members of the team to allow them to use the playbook to start an incident, as well as edit the contents.

## Editing a playbook

> You can edit a playbook that's being used in an active incident. However, the changes will only be applied to future incidents - not the active incident, or incidents that previously used that playbook.

Reviewer: I feel like we can remove the "that's being used in an active incident". Something like 
```suggestion
You can edit a playbook at any time. However, the changes will only be applied to future incidents - not the active incidents, or incidents that previously used that playbook.
```
